furious/async.py

```diff
@@ -91,6 +91,7 @@ def run_me(*args, **kwargs):
 MAX_DEPTH = 100
 MAX_RESTARTS = 10
 DISABLE_RECURSION_CHECK = -1
+RETRY_SLEEP_SECS = 4
```

Comment:
if you're going to make the retry logic configurable, you may want to make the retry time configurable... up to you.